### PR TITLE
Move webpack to peerDependencies

### PR DESCRIPTION
**Pull request by sapegin**

### CODE REVIEW

**Comment by andywer** [comment]
Already done by #169 and has been published as `v1.0.0-beta.2` 😉

**Comment by sapegin** [comment]
It’s the dev-server, not the main package.

**Comment by andywer** [comment]
Ooops, my bad 🙈

File: packages/dev-server/package.json [hunk 1]
```diff
@@ -19,9 +19,11 @@
   "repository": "andywer/webpack-blocks",
   "bugs": "https://github.com/andywer/webpack-blocks/issues",
   "dependencies": {
-    "webpack": "^2.6.1",
     "webpack-dev-server": "^2.4.5"
   },
+  "peerDependencies": {
+    "webpack": "2.x || 3.x"
+  }
```
Comment (andywer) [inline packages/dev-server/package.json]: Missing comma. Would have fixed it myself, but seems like you didn't grant me permission to edit files in the PR 😉

**Comment by andywer** [comment]
Thanks, @sapegin! :)

**Pull request merged**